### PR TITLE
feat:  Create deno.json config

### PR DESCRIPTION
**Pull request by beingminimal**
- Create deno.json config using deno 2.0

### CODE REVIEW

File: sdk/typescript/deno.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "unstable": [
+    "bare-node-builtins",
+    "byonm"
+  ]
+}
```
Comment (TomChv) [inline sdk/typescript/deno.json:6]: Where's the import map? Like in TypeScript with I'll do `import { request } from 'graphql-request'`, it's supposed to be `import { request } from 'npm:graphql-request` to work in deno?

Comment (beingminimal) [reply sdk/typescript/deno.json:6]: We don't need to maintain seperate import map file. We can directly use node_modules with deno by enabling below flags. 

**[byonm](https://docs.deno.com/runtime/reference/cli/unstable_flags/#--unstable-byonm)**
This feature flag enables support for resolving modules from a local node_modules folder that you manage outside of Deno with [npm](https://www.npmjs.com/), [pnpm](https://pnpm.io/), or [yarn](https://yarnpkg.com/).

**[bare-node-builtins](https://docs.deno.com/runtime/reference/cli/unstable_flags/#--unstable-bare-node-builtins)**
This flag enables you to [import Node.js built-in modules](https://docs.deno.com/runtime/reference/node/node_specifiers/) without a node: specifier

Comment (TomChv) [reply sdk/typescript/deno.json:6]: Nice!

File: sdk/typescript/package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "@grpc/grpc-js": "^1.11.1",
     "@lifeomic/axios-fetch": "^3.1.0",
     "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/core": "^1.26.0",
     "@opentelemetry/exporter-trace-otlp-grpc": "^0.52.1",
     "@opentelemetry/sdk-metrics": "^1.25.1",
```
[the remark this answers is not in the document]
Comment (beingminimal) [reply sdk/typescript/package.json:25]: Yes, we tried to add `deno run -A npm:mocha` but getting few errors.

[the remark this answers is not in the document]
Comment (TomChv) [reply sdk/typescript/package.json:25]: What are the error? They must be fixed

File: sdk/typescript/package.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "@grpc/grpc-js": "^1.11.1",
     "@lifeomic/axios-fetch": "^3.1.0",
     "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/core": "^1.26.0",
```
[the remark this answers is not in the document]
Comment (TomChv) [reply sdk/typescript/package.json:23]: Make sense, should be imported thanks to `@opentelemetry/api` but okay

File: sdk/typescript/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (beingminimal) [reply sdk/typescript/package.json:25]:
```
deno run test:deno

Task test:deno mocha
error: unexpected argument '--loader' found

  tip: a similar argument exists: '--reload'
  tip: to pass '--loader' as a value, use '-- --loader'

Usage: deno run --reload[=<CACHE_BLOCKLIST>...] [SCRIPT_ARG]...
```

[the remark this answers is not in the document]
Comment (TomChv) [reply sdk/typescript/package.json:25]: Just follow what's the error says, I'll do the change on monday.

**Comment by TomChv** [comment]
Closing that one since it's already available on the PR but it's broken

**Pull request closed without merging**